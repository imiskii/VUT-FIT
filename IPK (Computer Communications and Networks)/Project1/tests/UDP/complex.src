(+ (* 10 10) (* 10 10))
(* 1000000 1000000)
(+ 5 6.58)
(/ 65 8)