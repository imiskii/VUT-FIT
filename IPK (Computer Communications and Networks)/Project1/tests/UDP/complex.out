OK:200
OK:1e+12
OK:11.58
OK:8.125

